microsoft-edge/dev-videos/index.md

```diff
@@ -29,6 +29,30 @@ To add a new video:
 -->
 
 
+<!-- ====================================================================== -->
+## DevTools - Use your preferred language in DevTools
```

Comment:
```suggestion
## DevTools - Use your preferred coding language in DevTools
```

Reply:
The languages I'm referring to here are not programming languages (which is what the "coding" word you suggest adding makes me think of). These are languages you can translate the user interface to, like French, German, Chinese, etc.

@vchapel any idea on how to make this clearer?

Reply:
Oh, that is worded correctly then, I misunderstood